src/Windowing/Extensions/Silk.NET.Windowing.Extensions.WebGPU/WebGPUWindow.cs

```diff
@@ -0,0 +1,107 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Silk.NET.Core.Contexts;
+using Silk.NET.WebGPU;
+using Silk.NET.Windowing;
+
+namespace Silk.NET.Windowing.Extensions.WebGPU;
+/// <inheritdoc/>
```

Comment:
From where?

Reply:
idk, the dotnet format command put that there